cmd/aries-js-worker/src/agent-rest-client.js

```diff
@@ -457,9 +457,31 @@ const pkgs = {
             method: "POST",
         },
     },
-    context: {
-        Add: {
-            path: "/context/add",
+    ld: {
+        AddContexts: {
+            path: "/ld/context/add",
```

Comment:
imo we don't need add suffix - usually HTTP POST verb implicitly denotes that.

Reply:
Not directly related to this PR, but we need to remove /add suffix [here](https://github.com/hyperledger/aries-framework-go/blob/eec182deab9a345ad2c66dc58d169233edbcdaaa/pkg/controller/rest/jsonld/context/operation.go#L22) as well - we can create a new issue/PR.